Editor/Scripts/Analytics/MeshsyncAnalyticsHandlers.cs

```diff
@@ -146,14 +177,17 @@ struct SyncEventData {
                 syncMode = syncData.syncMode
             };
 
-            Send(eventData);
+            Send(eventData, entityTypeStr);
```

Comment:
Can you have both Asset and Entity types in data? what happens if entityType is none and Asset type has a value, should we not handle it and send the asset type instead?

Reply:
Yeah correct, we need to handle both asset type and entity type.

If you look at Colm's PR you should be able see what needs to be handled

Reply:
Yeah probably, fixed now